tests/integration/test_elle.py

```diff
@@ -0,0 +1,293 @@
+"""
+This file is part of RedisRaft.
+
+Copyright (c) 2020-2021 Redis Ltd.
+
+RedisRaft is licensed under the Redis Source Available License (RSAL).
+"""
+import time
+import pytest as pytest
+
+from .sandbox import ElleWorker, Elle, key_hash_slot
+
+
+@pytest.mark.skipif("not config.getoption('elle_threads')")
+def test_elle_sanity(created_clusters):
+    time.sleep(5)
+
+
+@pytest.mark.skipif("not config.getoption('elle_threads')")
+def test_elle_migrating_manual(elle, cluster_factory):
+    cluster1 = cluster_factory().create(3, raft_args={
+        'sharding': 'yes',
+        'external-sharding': 'yes'})
+    cluster2 = cluster_factory().create(3, raft_args={
+        'sharding': 'yes',
+        'external-sharding': 'yes'})
+
+    cluster1_dbid = cluster1.leader_node().info()["raft_dbid"]
+    cluster2_dbid = cluster2.leader_node().info()["raft_dbid"]
+    slot = key_hash_slot("test")
+
+    client_map = elle.map_addresses_to_clients([cluster1, cluster2])
+
+    worker = ElleWorker(elle, client_map)
```

Comment:
Was this test case for POC? Do we need to do things manual?

Reply:
I think there's value in doing tests not just as background to an existing test.  but am open to removing it.